openquake/server/templates/engine/get_outputs.html

```diff
@@ -15,11 +15,17 @@ <h2>Outputs from calculation {{ calc_id }}</h2>
                   Download hdf5 datastore</a>
               </div>
               {% if hmaps %}
-              <div id="my-hmaps">
-                <a href="{{ oq_engine_server_url }}/v1/calc/{{ calc_id }}/hmap_0_0" class="btn btn-sm">
+              <div class="my-pngs">
+                <a href="{{ oq_engine_server_url }}/v1/calc/{{ calc_id }}/download_png/hmap_0_0" class="btn btn-sm">
                   Show hazard map</a>
               </div>
               {% endif %}
+              {% if meanHCs_afe_RTGM %}
+              <div class="my-pngs">
+                <a href="{{ oq_engine_server_url }}/v1/calc/{{ calc_id }}/download_png/meanHCs_afe_RTGM" class="btn btn-sm">
+                  Show meanHCs_afe_RTGM</a>
```

Comment:
@kejohnso it is especially important here, because this is the label the user will see on the button to visualize the png